webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -299,9 +299,13 @@ trait NodeInfoServiceCached extends NodeInfoService with NamedZioLogger with Cac
       def getDataIO(lastKnowModification: DateTime): IOResult[AllNodeEntries] = {
         for {
           con     <- ldap
+          t0      = System.currentTimeMillis
```

Comment:
You need to use the 'current time millis' in our ZioCommon that returns an ioresult